src/reaction_lammps_mupt/reaction_template_builder/lunar_client/lunar_api_wrapper.py

```
@@ -241,72 +277,57 @@ def run_LUNAR_all2lmp(molecule_files_typed: dict[str, list[Path]]) -> dict[str,
-    # Add footer with output directory specification
-    merge_files += f""" 
-# Specify the parent_directory of where to write results (optional) 
-parent_directory = "{test_cache_bond_react_merge}"
+
+        full_file = normalize_path(Path(cache_all2lmp) / lammps_file)
+        merge_files += f"{name:<10}{full_file:<150}{comment}\n"
+
+    # Specify where the merge outputs should be written to within the merge_input file.
+    merge_files += f"""
+# Specify the parent_directory of where to write results (optional)
 """
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)